lib/middleware/request-handler.ts

```diff
@@ -87,7 +87,7 @@ export function elevateRequest (req: Interface.IOurRequest,
         });
     };
 
-    res.sendError = function(err, where, reason): Promise<void> {
+    res.sendError = function(err: any, where: string, reason: any): Promise<void> {
```

Comment:
Let's leave it for now for the purposes of this PR and then in a follow-up remove this.